File: cs/ex3_2_2.cs
class Chooser<T> {
    T v1;
    T v2;
    public T choose() { return true ? this.v1 : this.v2; }
}

class Example {
    public static void Main() {
        Chooser<int> c = new Chooser<int>();
        int n = c.choose();
    }
}

File: kotlin/ex3_2_2.kt
class Chooser<T>(val v1: T, val v2: T) {
    fun choose(): T = if (true) this.v1 else this.v2
}

fun main() {
    val c: Chooser<Int> = Chooser<Int>(1, 2)
    val n: Int = c.choose()
}
